### PR TITLE
adding flask route

### PR DESCRIPTION
**Pull request by Halix267**

### CODE REVIEW

**Comment by Halix267** [comment]
@ADI10HERO This is the change in app.py ..

File: dev_server/app.py [hunk 1]
```diff
@@ -1,14 +1,50 @@
...
-@app.route('/')
-def hello():
-    """Return Hello World."""
-    return "Hello World!"
+    
 
 
 if __name__ == '__main__':
-    app.run(port=5000, debug=True)
+    app.run(port=5000, debug=True)
```
Comment (ADI10HERO) [inline dev_server/app.py:50]:
```suggestion
    app.run(port=5000, debug=True)

```

**Comment by ADI10HERO** [comment]
give @Halix267 75 points

**Comment by ADI10HERO** [comment]
@shintan777 
Did you get a chance to reproduce and/or solve the issue?

**Comment by ADI10HERO** [comment]
Closing this as the issue is not relevant anymore. Thank you for your contribution.

**Pull request closed without merging**